tool/tsh/tsh.go

```diff
@@ -600,8 +603,10 @@ func Run(ctx context.Context, args []string, opts ...cliOption) error {
 		EnumVar(&cf.MFAMode, modes...)
 	app.HelpFlag.Short('h')
 
-	ver := app.Command("version", "Print the version of your tsh binary")
+	ver := app.Command("version", "Print the version of your tsh binary and version of the Teleport proxy in current tsh profile")
```

Comment:
For reference, `kubectl` uses the following help message:

```
$ kubectl help version
Print the client and server version information for the current context.
```

Can we do something similar?

Reply:
Updated to 

`Print the tsh client and Proxy server versions for the current context.`